src/leap/mail/imap/mailbox.py

```diff
@@ -639,21 +533,41 @@ def _do_fetch(self, messages_asked, uid, d):
 
         sequence = False
         # sequence = True if uid == 0 else False
```

Comment:
it's true, this is confusing as hell. I've renamed the bool to is_sequence (which is not supported right now, will need changes in mailbox_indexer), and I called the other "sequences" message_ranges